code/FulpstationCode/fulpspaceruins.dm

```diff
@@ -167,4 +227,11 @@ There is no point for a testament, but maybe a confession is appropriate. I've n
 	suffix = "prototype.dmm"
 	name = "Prototype SS13"
 	placement_weight = 3 //Fun spawners in space are rare. Might be toned down in the future.
-	description = "Apparently, the first station built by NanoStrasen, but the Syndicates may want to reclaim it."
\ No newline at end of file
+	description = "Apparently, the first station built by NanoStrasen, but the Syndicates may want to reclaim it."
+
+/datum/map_template/ruin/space/dndmansion
+	id = "dndmansion"
+	suffix = "DnDmansion.dmm"
+	name = "dndmansion"
+	placement_weight = 3 //Fun spawners in space are rare. Might be toned down in the future.
+	description = "a rich guy hired a bunch of nerds so he can watch you play dnd.That nerd is you. have fun."
```

Comment:
Capitalization.

Also, add a space after that first   P E R I O D , please.

Reply:
that punctuation mastake was actually my fault.